components/experimental/src/units/ratio.rs

```diff
@@ -22,29 +22,45 @@ use crate::measure::provider::si_prefix::{Base, SiPrefix};
 pub struct IcuRatio(Ratio<BigInt>);
 
 /// Represents an error when the ratio string is invalid and cannot be parsed.
-#[derive(Debug, PartialEq)]
+#[derive(Debug, PartialEq, displaydoc::Display)]
 pub enum RatioFromStrError {
     /// Represents an error when the ratio string is divided by zero.
     DivisionByZero,
 
     /// Represents an error when the ratio string contains multiple slashes.
+    ///
     /// For example, "1/2/3".
+    #[displaydoc("Represents an error when the ratio string contains multiple slashes")]
```

Comment:
Nit, here and elsewhere: keep the editorializing to docs. Say just "ratio string contains multiple slashes"

Reply:
https://github.com/unicode-org/icu4x/pull/5972